### ISSUE TITLE
ImageView set_enter_focus warning

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#465]
I am using the latest dev with `stginga`. Should I be worried about this warning?

```
2017-05-12 14:55:35,888 | W | ImageView.py:3061 (set_enter_focus) |
    Subclass should override this abstract method!
```

**Comment by ejeschke** [comment]
Is this with the qt back end?  I don't think you should see this with qt.  And in fact you probably shouldn't see it with most of the back ends either.  Most of them do override that method.

**Comment by pllim** [comment]
Yes, Qt5. I do have a bit of problem of weird things happening trying to run stuff installed using RHEL6 on  RHEL7 and vice versa. Although this might also have been happening in a while without me knowing since I often just set `loglevel=40`, not 30. I'll keep an eye out and report more if I see it again.

**Comment by pllim** [comment]
Update: I also see this when only using Ginga, not `stginga`.
```
$ ginga --loglevel=30 --stderr
2017-05-13 10:14:59,430 | W | ImageView.py:3061 (set_enter_focus) | Subclass should override this abstract method!
```

**Comment by pllim** [comment]
Changing the warning into an error (which is not a good idea, by the way), gives me the following traceback:
```
Traceback (most recent call last):
  File ".../bin/ginga", line 11, in <module>
    load_entry_point('ginga==2.6.4.dev1475', 'console_scripts', 'ginga')()
  File ".../ginga/rv/main.py", line 661, in _main
    reference_viewer(sys.argv)
  File ".../ginga/rv/main.py", line 656, in reference_viewer
    viewer.main(options, args)
  File ".../ginga/rv/main.py", line 539, in main
    ginga_shell.add_channel(chname)
  File ".../ginga/rv/Control.py", line 1112, in add_channel
    workspace=workspace)
  File ".../ginga/rv/Control.py", line 1615, in add_viewer
    fi = self.build_viewpane(settings, size=size)
  File ".../ginga/rv/Control.py", line 1582, in build_viewpane
    fi.set_enter_focus(settings.get('enter_focus', True))
  File ".../ginga/ImageView.py", line 3062, in set_enter_focus
    raise NotImplementedError("Subclass should override this abstract method!")
```

**Comment by ejeschke** [comment]
This is fixed in commit e30d389355d0684c90359e0858d55f741fe21967.  The order of a subclass and a mixin was switched to fix the MRO.

**Comment by pllim** [comment]
Yes. It is gone now. Thanks!
